tests/image_verify.sh

```diff
@@ -315,12 +315,21 @@ EXIT_STATUS=0
 curl -Ls -o ${BASE}/wait-for https://github.com/eficode/wait-for/releases/download/v${WAIT_FOR_VER}/wait-for && chmod +x ${BASE}/wait-for
 
 process_docker_env
-debug "Docker run command: docker run ${DOCKER_ENV} --rm -d -v ${PWD}/tests/php-test-scripts:/var/www/html ${DOCKER_IMAGE}"
-CONTAINER_ID=$(docker run ${DOCKER_ENV} --rm -d -v ${PWD}/tests/php-test-scripts:/var/www/html ${DOCKER_IMAGE})
+RUNCMD="docker run ${DOCKER_ENV} -d -v ${PWD}/tests/php-test-scripts:/var/www/html ${DOCKER_IMAGE}"
+debug "Docker run command: ${RUNCMD}"
+CONTAINER_ID=$(${RUNCMD})
 if [ $? -ne 0 ]; then
     echo "Failed to start the docker image"
     exit 9
 fi
+if [ ! $(docker ps | grep ${CONTAINER_ID}) ]; then
+    echo ""
+    echo "Failed to start the docker image, with the following errors:"
+    docker logs ${CONTAINER_ID}
+    echo ""
+    docker rm -vf ${CONTAINER_ID}
+    exit 9
+fi
```

Comment:
@Monska85 it seems that exit status is not enough because containers can start and then fail at the entrypoint, that is not a reponsability of the docker daemon, what i've added it an extra check to determine if container is really up and running.